src/routes/+layout.svelte

```diff
@@ -76,6 +81,10 @@
 <style lang="postcss">
 	:global(body) {
 		margin: 0;
+		background-color: var(--colors-base);
+	}
+	:global(a) {
+		color: var(--colors-top);
 	}
```

Comment:
I think I less and less like this here and would like to see it in a separate css. BTW there is also this syntax AFAIK.
```suggestion
	:global() {
		body {
				margin: 0;
				background-color: var(--colors-base);
		}
		a {
			color: var(--colors-top);
		}
	}
```

Reply:
This syntax causes compile errors.
Also using this syntax causes a linter error and it's better to be avoided:
```
.selector {
  :global(element) {
  /* ... */
  }
}
```